src/cct.c

```diff
@@ -371,14 +373,35 @@ int main(int argc, char **argv) {
         }
         proj_errno_restore (P, err);
 
+        /* handle comment string */
+        if (opt_given(o, "c")) {
+            /* what number is the last coordinate column in the input data? */
+            int colmax = 0;
+            for (i=0; i<4; i++)
+                colmax = MAX(colmax, columns_xyzt[i]);
+            comment = column(buf, colmax+1);
+        } else {
```

Comment:
The else block can be left out if introducing the "blank_comment" initialization above